exercises/01-Welcome/README.md

```diff
@@ -8,7 +8,8 @@ intro: "https://www.youtube.com/watch?v=GjQEotj3t6Y"
 
 This is the first in a series interactive tutorials meant to provide everything you need to start your journey into the world of Javascript.
 
-## You will learn: 
+## You will learn:
```

Comment:
this was a weird one...looked like a space after `You`, but it was some other character, so deleting and retyping fixed the markdown. I can remove the extra line added if you'd prefer to keep it directly under the title.